apps/http-server/packages/generic/src/packageVersion.ts

```diff
@@ -0,0 +1,5 @@
+/**
+ * The version of the package.json file
+ * Note: This variable is updated at build-time by scripts/prebuild.js
+ */
+export const PACKAGE_JSON_VERSION = '1.50.6'
```

Comment:
I still feel like this entire file should be gitignored, since it's generated.

Reply:
Good point! I've excluded it now